plugin-server/src/worker/ingestion/event-pipeline/2-emitToBufferStep.ts

```diff
@@ -104,7 +104,12 @@ export function shouldSendEventToBuffer(
         !!event.properties &&
         ['posthog-ios', 'posthog-android', 'posthog-react-native', 'posthog-flutter'].includes(event.properties['$lib'])
 
-    const sendToBuffer = !isMobileLibrary && !person && !isAnonymousEvent && !isIdentifyingEvent
+    const shouldBufferAnonymousEvents = hub.teamIdsToBufferAnonymousEventsFor.has(teamId)
+
+    const processEventImmediately =
+        isMobileLibrary || person || isIdentifyingEvent || (isAnonymousEvent && !shouldBufferAnonymousEvents)
```

Comment:
I hope we're not going to find out that mobile libs are the cause for most of the others when we run it on all teams